src/components/pages/JudgeTable/JudgeTable.js

```diff
@@ -68,17 +68,18 @@ export default function JudgeTable(props) {
   ];
 
   judgeData.forEach((item, idx) => {
-    item.id = idx; //no?
+    item.id = idx;
   }); // this is VERY hacky, but the table doesn't take data without ids
 
   const findRowByID = (rowID, rowData) => {
     for (let i = 0; i < rowData.length; i++) {
       let currentRow = rowData[i];
-      if (currentRow.name === rowID) {
+      let adjustedRowID = parseInt(rowID) + 1;
```

Comment:
This is some smart account for the difference in counting in the table and counting with JS on the backend - good work!